### PR TITLE
[W5.11][T09-A2] Goh Keng Seng

### PR DESCRIPTION
**Pull request by kengsengg**
Enhancement 2: Allows user to sort the list of people in address book in alphabetically ascending order (case insensitive)

### CODE REVIEW

**Comment by chenxing1992** [comment]
Why isnt your enhancement for both combined together?

**Review by chenxing1992 — COMMENTED** [review]
Well done ! codes are easy to understand!

File: src/seedu/addressbook/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import static seedu.addressbook.common.Messages.MESSAGE_LIST_SORTED;
+
+/**
+ * Sorts all the people in AddressBook by their names in alphabetically ascending order (case insensitive)
+ */
+
+public class SortCommand extends Command {
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sorts the entire list of people in AddressBook by their names in alphabetically ascending order (case insensitive).\n"
```
Comment (dalessr) [inline src/seedu/addressbook/commands/SortCommand.java:16]: Avoid lines longer than 110 chars. When a line is too long, it requires horizontal scrolling, that slows down the reader/reviewer.

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (dalessr) [inline src/seedu/addressbook/parser/Parser.java:14]: Avoid auto-formatting unrelated parts of the code. Doing so can result in unrelated changes being included in the commit. You can use sourcetree to look through changes before committing, and discard any unrelated changes.

This is a coding style violation, please avoid doing this in future.

**Review by dalessr — COMMENTED** [review]
Good job on covering all aspects of the new command @kengsengg  👍

Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**